.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  binaries:
+    strategy:
+      matrix:
+        goos: [linux]
+        goarch: [amd64, arm64, ppc64le, s390x]
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
```

Comment:
Should we fix a Go version here? You can use `go-version: 1.x`, or `go-version-file: go.mod` to read it from `go.mod`.